jenkins/open-mpi-build-script.sh

```diff
@@ -165,19 +165,28 @@ echo "--> Building commit ${sha1}"
 
 if test -f autogen.pl; then
     echo "--> running ./autogen.pl ${AUTOGEN_ARGS}"
-    ./autogen.pl ${AUTOGEN_ARGS}
+    if ! ./autogen.pl ${AUTOGEN_ARGS}; then
+        echo "./autogen.pl ${AUTOGEN_ARGS} failed, ABORTING !"
+        exit 1
+    fi
 else
     if test "${AUTOGEN_ARGS}" != ""; then
 	echo "--> Being a coward and not running with special autogen arguments and autogen.sh"
 	exit 1
     else
 	echo "--> running ./atogen.sh"
-	./autogen.sh
+	if ! ./autogen.sh; then
+            echo "./autogen.sh failed, ABORTING !"
+            exit 1
+        fi
     fi
 fi
 
 echo "--> running ./configure --prefix=\"${PREFIX}\" ${CONFIGURE_ARGS}"
-./configure --prefix="${PREFIX}" ${CONFIGURE_ARGS}
+if ! ./configure --prefix="${PREFIX}" ${CONFIGURE_ARGS}; then
```

Comment:
I'd combine the two commits and only change configure.

Reply:
Will do tomorrow